iothub/device/tests/Pipeline/RetryDelegatingHandlerImplicitOpenTests.cs

```diff
@@ -89,7 +86,7 @@ public async Task OpenAsync_SynchronizesToOneInnerCallToOpen()
             int callCounter = 0;
             var nextHandlerMock = new Mock<IDelegatingHandler>();
             nextHandlerMock
-                .Setup(x => x.OpenAsync(CancellationToken.None))
```

Comment:
Setup should expect "any" cancellation token and not only CancellationTokne.None.